### PR TITLE
[Tech]: update DependsOn README.md

### PR DESCRIPTION
**Pull request by liubou-masiuk**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md [hunk 1]
```diff
@@ -198,58 +197,57 @@ class MyComponent {
...
- - Multiple reference: `@@reference`. Starts from double @ symbols. Allows to access a group of field values marked by the same reference name.
- Multiple reference always returns array in the query.
+ - 'Single' reference: `@reference`. 'Single' reference starts from the `@` symbol in the Query, it allows you to access a defined field value.
+'Single' reference should reference an existing field and will not be reattached on dynamic DOM change.
+ - 'Multiple' reference: `@@reference`. Starts from double `@` symbols. Allows you to access a group of field values marked by the same reference name.
+ 'Multiple' reference always returns an array in the Query.
 
-Note: multiple reference triggers query update on any group update: changing some of group fields value, adding or removing referenced field.
-So usage of multiple reference can slow down queries performance.
+Note: 'multiple' reference triggers query update on any group update: changing some of group’s fields value or adding/removing a referenced field.
```
Comment (ala-n) [inline ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md:205]: `Note: 'multiple' reference triggers Query update on ...`

File: ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md [hunk 2]
```diff
@@ -102,17 +101,17 @@ Allowed reference types:
...
 
-In any other case (e.g. if type is `any`) no cast will be performed.
+In any other case (e.g. if the type is `any`), no cast will be performed.
 
 ##### ElementsAccessor Registry
 
-Registry `Granite.DependsOnPlugin.ElementAccessors` - can be used to define custom accessors of element.
-Accessor provide the information how to get/set value, set a require/visibility/disabled state or returns `preferableType` for specific type of component.
+Registry `Granite.DependsOnPlugin.ElementAccessors` - can be used to define the custom accessors of an element.
+The accessor provides information on how to get/set values and set a require/visibility/disabled state, or returns `preferableType` for the specific type of component.
```
[the remark this answers is not in the document]
Comment (liubou-masiuk) [reply ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md]: done

File: ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md [hunk 3]
```diff
@@ -1,90 +1,89 @@
...
+To define data attributes from JCR, use the _**granite:data**_ sub-node under the widget node.
 AEM Authoring Toolkit provides a set of annotations to use DependsOn from Java code.
 
-DependsOn workflow consists of the following steps:
+The DependsOn workflow consists of the following steps:
 
 **ObservedReference**  ─┐
 
-**ObservedReference**  ─── **QueryObserver[Query]***  ─── **Action***
+**ObservedReference**  ─── **QueryObserver[Query]**  ─── **Action**
```
[the remark this answers is not in the document]
Comment (liubou-masiuk) [reply ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md]: done

File: ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md (hunk 1 above)
Comment (liubou-masiuk) [reply ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md]: done

**Review by ala-n — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Review by smiakchilo — APPROVED** [review]

**Pull request merged**